src/containers/documentation/ApiOverviewPage.tsx

```diff
@@ -3,13 +3,38 @@ import * as React from 'react';
 import { Helmet } from 'react-helmet';
 import { Link, useParams } from 'react-router-dom';
 import ReactMarkdown from 'react-markdown';
+
 import { PageHeader } from '../../components';
+import { APIDescription } from '../../apiDefs/schema';
 import { APIUrlSlug } from '../../types';
+import { lookupCategoryByApi } from '../../apiDefs/query';
+
 import { getApi } from './DocumentationRoot';
+import './ApiOverviewPage.scss';
+
+const APITags = ({ api }: { api: APIDescription }): JSX.Element => {
+  const isCCG = api.oAuthTypes?.includes('ClientCredentialsGrant');
+  const isACG = api.oAuthTypes?.includes('AuthorizationCodeGrant');
+  const isOpen = api.openData;
+  const isRestricted = api.vaInternalOnly;
+
+  const apiCategory = lookupCategoryByApi(api);
```

Comment:
Replace `lookupCategoryByApi` with `lookupApiCategory`.
```suggestion
  const apiCategory = lookupApiCategory(api.categoryUrlFragment);
```